src/Exceptions/PageNameNotFoundExecption

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Mediconesystems\LivewireDatatables\Exceptions;
+
+use Exception;
+use JetBrains\PhpStorm\Internal\LanguageLevelTypeAware;
```

Comment:
Will this work for people that do not use PhpStorm ?

Reply:
Probably not, I'll remove.